applications/serial_lte_modem/src/nrf52_dfu/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_include_directories(.)
+target_sources_ifdef(CONFIG_SLM_NRF52_DFU app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/slm_at_dfu.c)
```

Comment:
This ifdef is redundant, this whole CMakeLists.txt file is sourced only if `CONFIG_SLM_NRF52_DFU` is enabled. Here and all below.